README.md

```diff
@@ -117,6 +132,7 @@ body, that says "hello".
 Your Answer:
 ```js
 // your code here
+$("#greeting").on("click",.append("<p>hello</p>"))
```

Comment:
So close, but you need to define wrap the callback in a function, and define what you're calling `append` on... (see solution for an example)